### DISCUSSION TITLE
Private package (Github registry) installation

### DISCUSSION THREAD

**Discussion by lmatheus** [vercel/vercel#4842] [Help]
Hello everyone,

I've tried the approach proposed on https://vercel.com/knowledge/using-private-dependencies-with-vercel by using my .npmrc

@username:registry=https://npm.pkg.github.com
//registry.npmjs.org/:_authToken=NPMTOKEN
//npm.pkg.github.com/:_authToken=GHTOKEN
Of course, replacing @username, NPMTOKEN and GHTOKEN with the relevant data and then running vc env add NPM_RC production < ~/.npmrc.

After I force a rebuild, the private package registry still points to the npm registry as opposed to github's and I get a 404 on that installation.

Anything you can think I can be missing here?

Thanks a lot in advance

**Comment by styfle** [comment]
You will probably need to add the env vars to both `production` and `preview`.

```sh
vc env add NPM_RC production < ~/.npmrc
vc env add NPM_RC preview < ~/.npmrc
```

You can check for existing env vars with `vc env ls` or in the project dashboard on vercel.com
